packages/block-library/src/audio/edit.native.js

```diff
@@ -153,46 +154,70 @@ function AudioEdit( {
 	}
 
 	return (
-		<>
-			<InspectorControls>
-				<PanelBody title={ __( 'Audio settings' ) }>
-					<ToggleControl
-						label={ __( 'Autoplay' ) }
-						onChange={ toggleAttribute( 'autoplay' ) }
-						checked={ autoplay }
-					/>
-					<ToggleControl
-						label={ __( 'Loop' ) }
-						onChange={ toggleAttribute( 'loop' ) }
-						checked={ loop }
-					/>
-					<SelectControl
-						label={ __( 'Preload' ) }
-						value={ preload || '' }
-						// `undefined` is required for the preload attribute to be unset.
-						onChange={ ( value ) =>
-							setAttributes( {
-								preload: value || undefined,
-							} )
-						}
-						options={ [
-							{ value: '', label: __( 'Browser default' ) },
-							{ value: 'auto', label: __( 'Auto' ) },
-							{ value: 'metadata', label: __( 'Metadata' ) },
-							{ value: 'none', label: __( 'None' ) },
-						] }
-					/>
-				</PanelBody>
-			</InspectorControls>
-			<MediaUpload
-				allowedTypes={ ALLOWED_MEDIA_TYPES }
-				isReplacingMedia={ true }
-				onSelect={ onSelectAudio }
-				render={ ( { open, getMediaOptions } ) => {
-					return getBlockUI( open, getMediaOptions );
-				} }
-			/>
-		</>
+		<TouchableWithoutFeedback
+			accessible={ ! isSelected }
+			onPress={ onAudioPress }
+			disabled={ ! isSelected }
+		>
+			<View style={ { flex: 1 } }>
```

Comment:
@ceyhun if we don't put this flex style here what would happen? I removed it and at first, I didn't notice anything. I know that it's a shorthand property to set default flexbox behavior. But that's as far as it goes 😄

Reply:
I added this, so the view fills 100% of its parent even though it's smaller than the available space. But for this case, I think we don't need it because I checked both [MediaUploadProgress](https://github.com/WordPress/gutenberg/blob/c183c3201d64e8a1d4920570901313a44ae5ebde/packages/block-editor/src/components/media-upload-progress/styles.native.scss#L2) (which is rendered by `MediaUpload`) and [BlockCaption](https://github.com/WordPress/gutenberg/blob/c183c3201d64e8a1d4920570901313a44ae5ebde/packages/block-editor/src/components/block-caption/styles.native.scss#L2) have `flex: 1` at top-level. 
We don't even need a `View` in this case. But `TouchableWithoutFeedback` supports only one child, and using a `View` is suggested [here](https://reactnative.dev/docs/touchablewithoutfeedback). For this reason, I'll keep the `View` but remove the `flex: 1`.

Reply:
Thanks for the explanation and change 👍